packages/elements/src/collapse/index.ts

```diff
@@ -185,24 +198,33 @@ export class Collapse extends BasicElement {
    * @returns clientHeight of the panel so that the panel holder max-height can be set
    */
   private getContentHeight (): number {
-    return this.panel && this.panel.clientHeight || 0;
+    const panelEl = this.panelRef.value;
+    return panelEl ? panelEl.clientHeight : 0;
   }
 
   /**
    * A `TemplateResult` that will be used
    * to render the updated internal template.
-   * @return {TemplateResult}  Render template
+   * @return Render template
    */
   protected render (): TemplateResult {
     return html`
-      <ef-header part="header" level="${this.level}" @tap="${this.handleTap}">
-        <ef-icon icon="right" slot="left" part="toggle"></ef-icon>
-        <slot slot="left" name="header-left"></slot>
-        <slot slot="right" name="header-right"></slot>
-        ${this.header}
+      <ef-header part="header" level="${this.level}">
+        <div part="heading" role="heading" aria-level="${ifDefined(this.headingLevel || undefined)}">
+          <div id="header-toggle"
+               part="header-toggle"
+               role="button"
+               tabindex="0"
+               aria-expanded="${this.expanded}"
+               aria-controls="content"
+               @tap=${this.toggle}>${this.header}</div>
+        </div>
+        <ef-icon icon="right" part="toggle" slot="left" aria-hidde="true"></ef-icon>
```

Comment:
typo `aria-hidden`. it makes the user can navigate to the icon.

Reply:
Fixed